src/constants/action-types.js

```diff
@@ -11,6 +11,7 @@
 export const ADD_FRAME = 'ADD_FRAME';
 export const UPDATE_GIF_PROGRESS = 'UPDATE_GIF_PROGRESS';
 export const ADD_GIF = 'ADD_GIF';
+export const UPDATE_GIF_FILENAME = 'UPDATE_GIF_FILENAME';
```

Comment:
@lopezh18 @phillipson808 -- take note of @colebillys19 action type here, and make sure your name is similar (e.g. `UPDATE_GIF_TITLE` or something

Reply:
Done